error.go

```diff
@@ -0,0 +1,13 @@
+package winrm
+
+import "fmt"
+
+// errWinrm generic error struct
+type errWinrm struct {
```

Comment:
Just for future use, I could update all errors to be this type instead of generic fmt.Errof().